buildenv/jenkins/variables/defaults.yml

```diff
@@ -403,12 +403,13 @@ x86-64_linux_valhalla:
 aarch64_linux:
   extends: ['debuginfo', 'freemarker', 'openjdk_reference_repo', 'openssl']
   boot_jdk:
+    8: '/usr/lib/jvm/java-1.8.0'
     11: '/usr/lib/jvm/adoptojdk-java-11'
   release:
+    8: 'linux-aarch64-normal-server-release'
```

Comment:
Is this correct? Other platforms don't contain "normal" in the name for 8.

Reply:
Other platforms contain"normal" for 8.
Example: https://github.com/eclipse/openj9/blob/dc7cbe4bddb2aa5b513b1c088dbb6aded66d4f4a/buildenv/jenkins/variables/defaults.yml#L193

Reply:
8 & 11 have "normal" it was removed in jdk12.

Reply:
Yup sorry, read it wrong.